testbed/tests/test_window.py

```diff
@@ -226,6 +226,28 @@ async def test_secondary_window_with_args(app, second_window, second_window_prob
 
         assert second_window not in app.windows
 
+    async def test_instantiate_window_with_content(app):
+        """A window can be created with content"""
+        try:
+            window_no_content = toga.Window()
+            assert window_no_content.content is None
+        finally:
+            window_no_content.close()
+            window_no_content = None
```

Comment:
As with the core APIs test - there's no need to test the "no content" case; that's tested elsewhere.